source/contribute.rst

```diff
@@ -64,8 +64,8 @@ ambiguous, we will add two explicit names for remote Git repositories:
 
 .. code-block:: console
     $ [`cd` into the directory created by `git clone ...`]
-    $ git remote add doctrine git@github.com:doctrine/<name-of-project>.git
-    $ git remote add fork git@github.com:<your-github-name>/<name-of-project>.git
+    $ git remote add doctrine git@github.com:doctrine/&lt;name-of-project&gt;.git
+    $ git remote add fork git@github.com:&lt;your-github-name&gt;/&lt;name-of-project&gt;.git
```

Comment:
@jaapio hi :wave: 

This is one of the buggy pages: https://www.doctrine-project.org/contribute/index.html

I think this is more like a rendering bug, wouldn't you agree? I don't think the source rst should be changed.

Reply:
Yes this looks like a rendering bug. The code blocks should have been escaped. And apparently they are not... Which is a security issue?

Reply:
Thanks for looking into it. They don't seem to do any HTML escaping indeed… but how would we replace these classes? Do you already have something built-in in phpdocumentor for rendering this kind of thing?

Reply:
https://github.com/doctrine/doctrine-website/pull/749 

Here you go, it's still in draft as I need to finish some missing things like aliases and it needs to be tested.

Reply:
Oh wow thanks!